features/product/managing_products/editing_product_slug.feature

```diff
@@ -8,7 +8,7 @@ Feature: Editing product's slug
         Given the store operates on a single channel in "United States"
         And I am logged in as an administrator
 
-    @ui @javascript @api
+    @ui @javascript @no-api
     Scenario: Creating a product with an autogenerated slug
```

Comment:
those scenarios is related to slug IMO it should be `no-api`

Reply:
right, I modified this scenario a bit to fit with the change in `ManagingProductContext` because we removed a setting price step so now this scenario should create a configurable product instead of simple